internal/common/utils.go

```diff
@@ -87,9 +87,9 @@ func CompareDevices(a models.Device, b models.Device) bool {
 
 func CompareDeviceProfiles(a models.DeviceProfile, b models.DeviceProfile) bool {
 	labelsOk := CompareStrings(a.Labels, b.Labels)
-	cmdsOk := CompareCommands(a.Commands, b.Commands)
+	cmdsOk := CompareCommands(a.CoreCommands, b.CoreCommands)
 	devResourcesOk := CompareDeviceResources(a.DeviceResources, b.DeviceResources)
-	resourcesOk := CompareResources(a.Resources, b.Resources)
+	resourcesOk := CompareResources(a.DeviceCommands, b.DeviceCommands)
```

Comment:
Should be fine this change:
`CompareCommands` → `CompareCoreCommands`
`CompareResources` → `CompareDeviceCommands`